tile-group.lisp

```diff
@@ -1052,6 +1052,39 @@ space."
             (show-frame-indicator group))
           (run-hook-with-args *remove-split-hook* l frame)))))
 
+(defcommand (remove-sibling tile-group)
+    (&optional (group (current-group))
+	       (frame (let ((window-type (type-of (current-window))))
+			           (if (eq window-type 'float-window)
+			               nil ;; (focus-all (current-window))
```

Comment:
This should be an unless form instead of if since the true case is nil

Reply:
This should be, yes. This was hastily tacked on at the last minute to replace a call to (current-frame), which I had implemented and forgotten wasnt a stumpwm function. this is why there is a commented out call to focus-all.